Assets/Plugins/Source/EOS_SDK/Generated/ContinuanceToken.cs

```diff
@@ -38,6 +38,18 @@ public Result ToString(out Utf8String outBuffer)
 			int inOutBufferLength = 1024;
 			System.IntPtr outBufferAddress = Helper.AddAllocation(inOutBufferLength);
 
+			// NOTES:
+			// 
+			// The changes below are implemented temporarily by PEW to correct
+			// an issue where the inOutBufferLength is not properly determined.
+			// 
+			// Until this is resolved from Epic - these changes will need to
+			// be maintained after the import of a new version of the EOS SDK.
+
+			// Call the EOS_ContinuanceToken_ToString function will a null buffer address to determine the correct buffer length.
```

Comment:
nit: "Call the EOS_ContinuanceToken_ToString function _with_ a null buffer address..." instead of "_will_".

Reply:
We have a notation for marking a 'PEW' specific modifications to the code:
```
//PEW: start modify
.....
//PEW: end modify
```

This is done to make it easy to kind of `grep` though the code to find PEW edits to the SDK without having to dig though git too much. I will lament that this notation isn't documented anywhere.